docs/modding/4-spriting.md

```diff
@@ -36,23 +36,23 @@ It is highly recommended that you use spriting software that supports transparen
       - Pretty bloated for spriting in Mindustry style.
 
   5. **[Paint.NET](https://www.getpaint.net/)**
-      - Very basic painting software, not to be confused with Paint 3D, Paint&#46;NET is usable but not as convenient as the above mentioned.
-      - Paint&#46;NET lacks basic features used for spriting in mindustry style. However, you can get some of these missing features with plugins that you can install manually.
```

Comment:
There's a reason that I did this, markdown would convert regular Paint.NET to a link,

Reply:
oh?